docs/pbc/all/multi-factor-authentication/202505.0/install-email-multi-factor-authentication-method.md

```diff
@@ -0,0 +1,442 @@
+cms-block-email--multi_factor_auth_email--text,customer_multi_factor_auth_email--text,TEXT Email Template With Header And Footer,@CmsBlock/template/email-template-with-header-and-footer.text.twig,1,,,,,,,"{{ 'mail.customer.multi_factor_auth.email.subject' | trans }}  {{ 'customer.multi_factor_auth.email.text' | trans({'%code%': mail.multiFactorAuth.multiFactorAuthCode.code}) }}","{{ 'mail.customer.multi_factor_auth.email.subject' | trans }}  {{ 'customer.multi_factor_auth.email.text' | trans({'%code%': mail.multiFactorAuth.multiFactorAuthCode.code}) }}"
+{% endraw %}
+```
+
+</details>
+
+## 4) Set up behavior
+
+### 4.1) For Customers
```

Comment:
Glue?

Reply:
Please see `4.4) For Glue Rest API`
There is a bit different namespace so I decided to move it into a separate section